src/components/Tabs.react.js

```diff
@@ -229,6 +320,69 @@ export default class Tabs extends Component {
+                // TODO: handle components that are not dcc.Tab components (throw error)
+                // enhance Tab components coming from Dash (as dcc.Tab) with methods needed for handling logic
+                let childProps;
+                const child = this.parseChildrenToArray()[0];
+                const length = this.props.children.length;
+                let currentIndex = 0;
+
+                const nextTabHandler = () => {
+                    currentIndex = this.props.value.slice(-1) - 1;
+                    currentIndex = ((currentIndex + 1) % length) + 1;
+                    this.props.setProps({value: `tab-${currentIndex}`});
+                };
+
+                if (
+                    // disabled is a defaultProp (so it's always set)
+                    // meaning that if it's not set on child.props, the actual
+                    // props we want are lying a bit deeper - which means they
+                    // are coming from Dash
+                    isNil(child.props.disabled) &&
+                    child.props._dashprivate_layout &&
+                    child.props._dashprivate_layout.props
+                ) {
+                    // props are coming from Dash
+                    childProps = child.props._dashprivate_layout.props;
+                } else {
+                    // else props are coming from React (Demo.react.js, or Tabs.test.js)
+                    childProps = child.props;
+                }
+
+                return (
+                    <div key={length}>
+                        <NextTab
+                            key={this.props.children.length}
+                            selectHandler={nextTabHandler}
+                            className={childProps.className}
+                            style={
+                                isNil(this.props.next_tab_layout.style)
+                                    ? childProps.style
+                                    : this.props.next_tab_layout.style
+                            }
+                            selectedClassName={childProps.selected_className}
+                            selected_style={childProps.selected_style}
+                            value={childProps.value}
+                            disabled={childProps.disabled}
+                            disabled_style={childProps.disabled_style}
+                            disabled_classname={childProps.disabled_className}
+                            mobile_breakpoint={this.props.mobile_breakpoint}
+                            vertical={this.props.vertical}
+                            colors={this.props.colors}
+                            loading_state={childProps.loading_state}
+                            title={this.props.next_tab_layout.title}
+                        />
+                    </div>
+                );
+            };
+        }
+
+        if (this.props.next_tab) {
+            EnhancedTabs = EnhancedTabs.concat(nextTab());
```

Comment:
Since it's the exact opposite, I think we should consider adding `Previous` as part of this PR.